third_party/amd/python/test/test_scalarize_vector_fops.py

```diff
@@ -0,0 +1,101 @@
+import re
```

Comment:
This is very ad-hoc test "infra". What about just doing lit test on llvm ir? Is it not working / too complicated / etc.?

Reply:
It's not possible - the pass is added in the python extension right? Recall I tried to add a test in the last PR that connected python and FileCheck and that was too complicated. On the otherhand if I added the pass to `triton-llvm-opt` I would have to reconstruct the entire pipeline we currently have in `compiler.py`. Note also I didn't "invent" this - there are lots of tests already that look like this https://github.com/triton-lang/triton/blob/bb60d9efe15534c1d0514e19cdf5da14d38779a2/python/test/unit/language/test_compile_only.py#L58

Reply:
Okay yeah it's fine then.